cookbooks/creating_a_behat_extension.rst

```diff
@@ -0,0 +1,278 @@
+Creating a Behat extension
```

Comment:
Now this is specific to initialising contexts, I agree with the suggestion to rename it.

Maybe call the file `creating_a_context_configuration_extension.rst` (and update the link in cookbooks.rst) and then change the title to something like:

```suggestion
Creating a Behat extension to provide custom configuration for Contexts
```

Reply:
Applied ✅  and file renamed